monitor.cpp

```diff
@@ -4,35 +4,66 @@
 #include <chrono>
 #include <iostream>
 using std::cout, std::flush, std::this_thread::sleep_for, std::chrono::seconds;
-
+/// @brief displayWarning function to display warning message
+void displayWarning() {
+  for (int i = 0; i < 6; i++) {
+    cout << "\r* " << flush;
+    sleep_for(seconds(1));
+    cout << "\r *" << flush;
+    sleep_for(seconds(1));
+  }
+}
+/// @brief to check if the vitals are ok
+/// @param temperature
+/// @param pulseRate
+/// @param spo2
+/// @return 1 if vitals are ok, 0 otherwise
 int vitalsOk(float temperature, float pulseRate, float spo2) {
+  if (checkVitals(temperature, pulseRate, spo2)) {
+    return 1;
+  }
+  return 0;
+}
+
+bool checkVitals(float temperature, float pulseRate, float spo2) {
```

Comment:
Suggest to rename all bool-return functions with "is" in the name. E.g., `allVitalsInRange`. Otherwise, `checkVitals` could mean "return true when vitals are in range" or "return true when attention is needed (vitals are out of range)"
The comments do make it clear, but I would need to refer to the comments every time I see a call to the function - a slight overhead